apps/browser/src/browser/browserApi.ts

```diff
@@ -227,10 +227,12 @@ export class BrowserApi {
     }
   }
 
-  static reloadOpenWindows() {
+  static reloadOpenWindows(exemptCurrentHref = false) {
+    const currentHref = window.location.href;
     const views = chrome.extension.getViews() as Window[];
     views
       .filter((w) => w.location.href != null && !w.location.href.includes("background.html"))
+      .filter((w) => exemptCurrentHref && w.location.href !== currentHref)
```

Comment:
Won't this filter out all views if `exemptCurrentHref` has its default value of `false`? No item would satisfy the predicate.

```suggestion
      .filter((w) => !exemptCurrentHref || w.location.href !== currentHref)
```

Reply:
Also, should we compare `windowId`s instead? That seems more robust to me, given that the href is not guaranteed to be unique. You actually want to exempt the current window, not all windows at the given href.